src/testoptions.c

```diff
@@ -96,18 +96,6 @@ void findCwndPeaks(tcp_stat_agent* agent, CwndPeaks* peaks,
   prevCWNDval = CurCwnd;
 }
 
-/**
- * Print the appropriate message when the SIGALRM is caught.
- * @param signo The signal number (shuld be SIGALRM)
- */
-
```

Comment:
Could you expand a little on mention in the PR description explaining what happened to the "other" alarm handler.
Also consider adding somewhere a comment block explaining how alarms should be used (or not) throughout NDT, and referencing it anywhere alarm() is called.